rulebook.tex

```diff
@@ -1537,6 +1537,7 @@ \subsection{Task Fulfillment and Scoring}
         Finish $C_2$ pre-cap & Mount the last ring of a $C_2$ product & $+30$ \\
         Finish $C_3$ pre-cap & Mount the last ring of a $C_3$ product & $+80$ \\
         Mount cap & Mount the cap on a product & $+10$ \\
+        Retrive cap & Buffer a cap into a cap station & $+2$\\
```

Comment:
Typo here, should be "Retrieve"

Reply:
Done..Thanks for noticing

Reply:
I don't see the fix?

Reply:
@MostafaGomaa can you please double-check, the typo is still there.

Reply:
```suggestion
        Retrieve cap & Buffer a cap into a cap station & $+2$\\
```

Reply:
No clue what went wrong there, my bad!
commited as fixup
Thanks^^